docs/01-get-started/installation.mdx

```diff
@@ -1,20 +1,236 @@
+project, and create a `.env` file with essential configurations. Here are some
+examples:
+
+````mdx-code-block
+<Tabs>
+<TabItem value="Minimal install">
+Use Front-Commerce without any extension: only the default theme with static data.
+Useful for evaluation purposes, but quickly limited.
+
+```typescript title="front-commerce.config.js"
+import { defineConfig } from "@front-commerce/core/config";
+import themeChocolatine from "@front-commerce/theme-chocolatine";
+import storesConfig from "./app/config/stores";
+import cacheConfig from "./app/config/caching";
+
+export default defineConfig({
+  extensions: [themeChocolatine()],
+  stores: storesConfig,
+  cache: cacheConfig,
+  configuration: {
+    providers: [],
+  },
+});
+```
+
+Finally, create a `.env` file with the following content:
+
+```shell title=".env"
+# Front-Commerce configuration
+FRONT_COMMERCE_ENV=development
+FRONT_COMMERCE_URL=http://localhost:4000
+FRONT_COMMERCE_HOST=localhost
+FRONT_COMMERCE_PORT=4000
+FRONT_COMMERCE_COOKIE_PASS=cookie-secret
+FRONT_COMMERCE_SITEMAP_TOKEN=sitemap-secret
+FRONT_COMMERCE_CACHE_API_TOKEN=cache-api-secret
+```
+
+</TabItem>
+<TabItem value="Magento2">
+   Use Front-Commerce with a headless Magento2 instance.
+
+```typescript title="front-commerce.config.js"
+import { defineConfig } from "@front-commerce/core/config";
+import themeChocolatine from "@front-commerce/theme-chocolatine";
+import magento2 from "@front-commerce/magento2";
+import storesConfig from "./app/config/stores";
+import cacheConfig from "./app/config/caching";
+
+export default defineConfig({
+  extensions: [magento2(storesConfig), themeChocolatine()],
+  stores: storesConfig,
+  cache: cacheConfig,
+  configuration: {
+    providers: [],
+  },
+  v2_compat: {
+    useApolloClientQueries: true,
+  },
+});
+```
+
+:::info
+
+Learn more about
+[configuring the Magento2 extension](/docs/remixed/extensions/magento2/) and the
+prerequisites.
+
+:::
+
+```shell title=".env"
+# Front-Commerce configuration
+FRONT_COMMERCE_ENV=development
+FRONT_COMMERCE_URL=http://localhost:4000
+FRONT_COMMERCE_HOST=localhost
+FRONT_COMMERCE_PORT=4000
+FRONT_COMMERCE_COOKIE_PASS=cookie-secret
+FRONT_COMMERCE_SITEMAP_TOKEN=sitemap-secret
+FRONT_COMMERCE_CACHE_API_TOKEN=cache-api-secret
+
+# Magento2 configuration (see https://developers.front-commerce.com/docs/remixed/extensions/magento2#configuration)
+FRONT_COMMERCE_MAGENTO2_ENDPOINT=https://magento2.example.com
+FRONT_COMMERCE_MAGENTO2_CONSUMER_KEY=xxxxxxxxx
+FRONT_COMMERCE_MAGENTO2_CONSUMER_SECRET=xxxxxxxxx
+FRONT_COMMERCE_MAGENTO2_ACCESS_TOKEN=xxxxxxxxx
+FRONT_COMMERCE_MAGENTO2_ACCESS_TOKEN_SECRET=xxxxxxxxx
+```
+
+</TabItem>
+</Tabs>
+
+````
+
+## Start the application
+
+You should now be able to start the application without errors by running the
+`start` command using your favorite package manager (we recommend
+[pnpm](https://pnpm.io/)):
+
+```
+pnpm start
+```
+
+🎉 Congrats, you should now see this screen!
```

Comment:
**thought:** `pnpm start` does not open a browser on my side.

Reply:
I added a link to localhost.